src/common/constants/generic-store-messages-types.ts

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-export class GenericStoreMessageTypes {
-    public static readonly storeStateChanged: string = 'STORE_CHANGED';
+export enum GenericStoreMessageTypes {
```

Comment:
I wouldn't recommend using a TypeScript enum for a type that has to pass over a browser serialization boundary; even if it does work, I don't want every maintainer that thinks about that to have to wonder "wait, does this work?"

I'd recommend just using a string constant, like what `src/common/messages.ts` does. (I think moving this to `messages.ts` would be reasonable, if you wanted)